docs/source/news.rst

```diff
@@ -21,6 +21,8 @@ Details about the build system unification can be found in :ref:`RFC7`.
 Updates
 -------
 
+* Docs: add new page Usage/Ellipsoids to explain the usage of ellipsoidal parameters
```

Comment:
We generally do not include doc updates in the NEWS section (unless they are quite significant) as they should be online at about the same time as they are merged.